gulpfile.babel.js

```diff
@@ -23,6 +23,7 @@ const nodeResources = (() => {
   return _.chain(result.toString().trim().split(/\r?\n/))
     // remove app root path
     .tap((r) => r.shift())
+    .filter((dep) => dep.match(/node_modules/g).length == 1)
```

Comment:
`--depth` doesn't play well with `--parseable`. It does, however, work with `--json`, but that adds extra overhead that probably isn't necessary.